docs/orchestrating-elastic-stack-applications/elasticsearch/snapshots.asciidoc

```diff
@@ -17,6 +17,8 @@ To set up automated snapshots for Elasticsearch on Kubernetes you have to:
 
 The examples below use the https://www.elastic.co/guide/en/elasticsearch/plugins/7.17/repository-gcs.html[Google Cloud Storage Repository Plugin].
 
+NOTE: Support for S3, GCS and Azure repositories is bundled in Elasticsearch by default from version 8.0.
```

Comment:
```suggestion
NOTE: Support for S3, GCS and Azure repositories is bundled with Elasticsearch by default from version 8.0.
```
I know it's `in` in the official Elasticsearch documentation. But my gut feeling is it should be `with`, maybe a native speaker can help.

Reply:
Both are currently used in the official docs.

Reply:
Let's stay aligned with the ES doc :man_shrugging: